web/src/components/SubscriptionLogo.jsx

```diff
@@ -0,0 +1,7 @@
+import subscribeIcon from "../assets/subscribeIcon.svg?url";
+
+function SubscriptionLogo() {
+	return <img src={subscribeIcon} alt="subscription-logo" />;
```

Comment:
The alt text of "subscription-logo" won't read very well in a screen reader. It will say "subscription hypen logo". Perhaps a better alt text might be "CYF Slack Project" or arguably no alt text at all.